src/_doesCrossjoinYieldAllCombinationsOfMembers.ts

```diff
@@ -23,7 +24,11 @@ export function _doesCrossjoinYieldAllCombinationsOfMembers(
         levelInCrossjoin.dimensionName === dimensionName &&
         levelInCrossjoin.hierarchyName === hierarchyName &&
         levelInCrossjoin.levelName === levelName &&
-        !isMdxCompoundIdentifier(levelInCrossjoin.match)
+        !(
+          isMdxCompoundIdentifier(levelInCrossjoin.match) &&
+          getSpecificCompoundIdentifier(levelInCrossjoin.match, { cube })
+            ?.type === "member"
+        )
```

Comment:
I would find it both more readable and safeproof like this:
```suggestion
        (!isMdxCompoundIdentifier(levelInCrossjoin.match) || getSpecificCompoundIdentifier(levelInCrossjoin.match, { cube })?.type === "level)
```